src/APES.jl

```diff
@@ -13,7 +13,7 @@ J. Li and P. Stoica. An adaptive filtering approach to spectral estimation and s
 Y. Hua, A. Gershman, and Q. Cheng. High-resolution and robust signal processing. Signal Processing and Communications. CRC Press, October 2003.
 
 """ ->
-function apes{T <: FloatingPoint}(y::Array{T}, f::Union(AbstractArray{T}, T); M::Int=int(length(y)/2), Q::Union(Array{T, 2}, Nothing)=nothing)
+function apes{T <: FloatingPoint}(y::Array{T}, f::Union(AbstractArray{T}, T); M::Int=Int(length(y)/2), Q::Union(Array{T, 2}, Nothing)=nothing)
```

Comment:
`Int` is not a constructor on Julia 0.3, and on 0.4 it will throw on a non-Integer, i.e., if `length(y)` is odd. `int` still works on Julia 0.4 and isn't doesn't throw a deprecation warning yet, but it rounds to nearest even, which may not be what you want. Instead you might prefer `div(length(y), 2)`.